### DISCUSSION TITLE
[FEAT] Stop Altering and Adding Obstacles to Flow of Assets and Tokens

### DISCUSSION THREAD

**Discussion by shiloh92** [sunflower-land/sunflower-land#4021] [Ideas]
## Summary of Request

The current state of this game resembles Million on Mars prior to its collapse, which as you may know no longer exists as a game. (Sales are abysmal due to a series of haphazard decisions by the devs and from decision making that resulted in an unplayable, uninteresting product)

Allow me to explain to the Sunflower Land devs why their decision-making-from-the-purse is shooting themselves in the foot.

1) By restricting normal users and bot users from withdrawing assets / tokens you are cutting the natural flow of interest toward your blockchain game.
When a blockchain game ceases to be a blockchain game, ending all traffic of its token (the measure of interest) and its assets or NFTs (the tangible result of labor/interest) it is like placing a dam in front of a free flowing river.
As players realize their efforts are in vain, they will lose interest. Blockchain games are not like traditional video games - people expect tangible returns of assets or tokens in blockchain games.
2) Real players will tell other players not to play it
Real, motivated blockchain players will spread word of mouth about the decisions of the devs. This will accelerate the collapse of natural flow of assets and tokens in and outside the Sunflower Land community.
3) The most ardent players, most devoted players will find further haphazard decisions to protect the "token" or "Sunflower Land" community are nothing more than the devs looking out for their bottom line, not the game's fun or the blockchain aspect of "own your assets and tokens" and carry them with you wherever  you please.

As a result of the latest changes to "proof humanity" of your players, this solidifies my perspective that nothing will change and new players should seriously beware of wasting any of their hard earned MATIC or time on this not-blockchain-in-spirit game.

I urge the devs to turn back before the entire "river" ceases, as further restrictions will end not only casual players converting to regular players but your ardent fans will also abandon the game.

Don't fix what ain't broke. Bots exist. So do real players that love to swap tokens. That's blockchain games for you, if that doesn't interest the devs and they wish to have a chokehold on people's sovereign assets and tokens, they should stick to Steam games.

Provide summary of the problem, and the value that is added if the feature was added (i.e. As a player, I want a way to do this in game, so that value can be added to me in this way).

## Proof of Community Demand
 
The community will cease if restrictions on blockchain sovereignty are implemented beyond the most recent changes. MoM (Million on Mars) is a living example of your current decision making and will end Sunflower Land as a community economy.

**Comment by Jericho-rpc** [comment]
Totally agree with you, the fact is that there is a gradual loss of some loyal players, they invested a lot of time and money, but there is no corresponding return, the developers can spam spam spam spam spam nft at will, do not consider the consequences of the pursuit of a momentary heat, in my opinion the game will gradually die in the long run, low fun without economy
